packages/twenty-front/src/pages/settings/developers/webhooks/components/SettingsDevelopersWebhookDetail.tsx

```diff
@@ -1,7 +1,15 @@
 import styled from '@emotion/styled';
-import { useState } from 'react';
+import { IconHandClick, IconNorthStar } from '@tabler/icons-react';
```

Comment:
@martmull couldnt find this icons on 'twenty-ui' icon exports,  lint is complaining

Reply:
```suggestion
import { IconHandClick, IconNorthStar } from 'twenty-ui';
```
if Icons are missing, add them in `packages/twenty-ui/src/display/icon/components/TablerIcons.ts`

Reply:
Got it! Thanks